suites/quincy/cephadm/customer_bz/tier-1_reduce_cephadm_log_spam.yaml

```diff
@@ -0,0 +1,77 @@
+#===============================================================================================
```

Comment:
Why separate test suite ? Can't it be part of any existing ?

Reply:
This is part of a customer BZ automation and polarion has tier 1 hence added it under this suite.

Reply:
Moved it under https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephadm/tier-1_5-1_service-apply-spec.yaml